vars/parseStageInfo.groovy

```diff
@@ -144,6 +144,10 @@ def call(Map config = [:]) {
         result['pragma_suffix'] = '-hw-medium'
       }
     }
+    if (stage_name.contains('with Valgrind')) {
+      result['ftest_arg'] = ''
```

Comment:
Why are you overriding `result['ftest_arg']`?  In what situation is it's value wrong in the context of adding valgrind to (some) functional testing?

Reply:
`ftest_arg` shouldn't be overwritten.  d704aad6672018107ebf92ba38a1bebff477dbda